### ISSUE TITLE
ArrayItemRemoval configuration doesn't support the "ignore" property

### ISSUE THREAD

**Issue by jakzal** [infection/infection#698]
| Question    | Answer
| ------------| ---------------
| Infection version | 0.13.1
| Test Framework version | 8.0.0
| PHP version | 7.3.2
| Platform    | MacOS
| Github Repo | -

Given the following config:

<details>
 <summary>infection.json.dist</summary>
 
 ```json
{
    "timeout": 2,
    "source": {
        "directories": [
            "src"
        ]
    },
    "logs": {
        "text": "build/infection-log.txt"
    },
    "mutators": {
        "@default": true,
        "ArrayItemRemoval": {
            "ignore": [
                "Foo\\Bar::baz"
            ]
        },
        "IdenticalEqual": false,
        "NotIdenticalNotEqual": false
    }
}
 ```
</details>

infection fails with the following exception:

<details>
 <summary>infection output</summary>
 
 ```
In JsonValidationException.php line 15:

  "infection.json.dist" does not match the expected JSON schema:
   - mutators.ArrayItemRemoval : Object value found, but a boolean is required
   - mutators.ArrayItemRemoval : The property ignore is not defined and the definition does not allow additional properties
   - mutators.ArrayItemRemoval : Failed to match at least one schema
```
</details>

I expected it to ignore the `ArrayItemRemoval` mutant in the specified method.

<details>
 <summary>phpunit.xml</summary>
 
 ```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.0/phpunit.xsd"
         bootstrap="vendor/autoload.php"
         beStrictAboutOutputDuringTests="true"
         beStrictAboutTodoAnnotatedTests="true"
         colors="true"
         verbose="true">

    <testsuites>
        <testsuite name="default">
            <directory suffix="Test.php">tests</directory>
            <exclude>tests/phar</exclude>
        </testsuite>
    </testsuites>

    <filter>
        <whitelist processUncoveredFilesFromWhitelist="true">
            <directory suffix=".php">src</directory>
        </whitelist>
    </filter>

    <logging>
        <log type="coverage-html" target="build/coverage" lowUpperBound="50" highLowerBound="95"/>
        <log type="coverage-clover" target="build/coverage.xml"/>
    </logging>
</phpunit>
 ```
</details>

Re #649

**Comment by borNfreee** [comment]
Released as [0.13.2](https://github.com/infection/infection/releases/tag/0.13.2).

Thanks @jakzal for the very detailed bugreport.

**Comment by jakzal** [comment]
Thank you @borNfreee. Works like a charm :)
